### ISSUE TITLE
Severe memory leak in xxxToStr callback resolvers

### ISSUE THREAD

**Issue by GoogleCodeExporter** [TES5Edit/TES5Edit#140]
```
What steps will reproduce the problem?
1. Open 001058FC in FNVEdit, open task manager
2. Scroll down to conditions where quest objectives are being resolved
3. Keep scrolling up and down, or just move mouse over conditions to force 
redraw and thus resolving

What is the expected output?
Consumed memory to stay the same.

What do you see instead?
Huge memory leak, like several megabytes every second of scrolling

What version of the product are you using? On what operating system?
Tracked it down to old FNVEdit 3.0.21, I guess earlier version has this too. 
All later version including svn are affected, including FO3Edit and TES5Edit.
Win7 x64

Please provide any additional information below.
Basically it happens inside all custom wbCTDAXXXXXToStr callbacks on 
MainRecord.ElementByName['Objectives'] line. I've stumbled on it in TES5Edit 
when doing similar wbXXXXAliasToStr resolvers.
```

Original issue reported on code.google.com by `zila...@gmail.com` on 19 Jul 2013 at 7:48
